test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -202,6 +305,19 @@ public class IrisVectorData
             public float[] PetalInfo;
         }
 
+        public class IrisVectorDataWithLoadColumnName
+        {
+            public int Label;
+
+            [LoadColumnName("SepalLength", "SepalWidth")]
```

Comment:
Not sure I understand this. What if you have hundreds of columns to be loaded as a single vector. Do you need to specify all the column names? Can't you specify a column index range?

Reply:
What would it mean to have a name and range? Why not just use the more efficient index based LoadColumn attribute at that point?

Reply:
Sorry, that's what I mean, to use the LoadColumn attribute for those cases. I was thinking about the `LoadColumn` attribute. I didn't realize that here you are using `LoadColumnName`, not the same attribute, ok. ;)